vercel.json

```
@@ -0,0 +1,12 @@
+{
+  "rewrites": [
+    {
+      "source": "/(.*)",
+      "destination": "/api"
+    }
+  ],
+  "outputDirectory": "build",
+  "cleanUrls": true,
+  "installCommand": "git submodule update --init --recursive && yarn install",
+  "buildCommand": "yarn build:all"
```

```suggestion
  "installCommand": "yarn install",
  "buildCommand": "yarn build:production"
```

I added the `build:production` script in the last workflow refactor. It will checkout the submodules for you, and it will set up the build to include production plugins.